resharper/test/src/CSharp/Intentions/QuickFixes/CachePropertyValueQuickFixTests.cs

```diff
@@ -0,0 +1,26 @@
+using JetBrains.ReSharper.FeaturesTestFramework.Intentions;
+using JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.QuickFixes;
+using NUnit.Framework;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Tests.CSharp.Intentions.QuickFixes
+ {
+     [TestUnity]
+     public class CachePropertyValueQuickFixTests : QuickFixTestBase<CachePropertyValueQuickFix>
```

Comment:
Please add tests with conditional accesses.
I'm especially wary of cases where conditional accesses don't match between merged expressions i.e. when `t?.position` and `t.position` are mixed.